medication statements/README.md

```diff
@@ -0,0 +1,363 @@
+# Medication Statement Mapping
+
+## XML HL7 > JSON FHIR
+
+A FHIR `MedicationStatement` is mapped from a XML HL7 `MedicationStatement` containing an `EhrSupplyAuthorise` component.
+
+When `ehrSupplyAuthorise` is referenced, it refers to `MedicationStatement / component [@typeCode="COMP"] / ehrSupplyAuthorise`.
+
+When `ehrSupplyDiscontinue` is referenced, it refers to find the first matching `ehrSupplyDiscontinue` where `ehrSupplyDiscontinue / reversalOf / priorMedicationRef / id [@root]` matching `MedicationStatement / component [@typeCode="COMP"] / ehrSupplyAuthorise / id [@root]` in the `EhrExtract`.
+
+When `ehrSupplyPrescribe` is referenced, it refers to `MedicationStatement / component [@typeCode="COMP"] / ehrSupplyPrescribe`.
+
+When `EhrExtract` is referenced, it refers to the parent `EhrExtract` in the XML.
+
+When `PracticeCode` is referenced, it refers to the losing practice ODS code.
+
+| Mapped to (JSON FHIR Medication Statement field) | Mapped from (XML HL7 / other source)                                                                                                                                       |
+|--------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id                                               | `ehrSupplyAuthorise / id [@root]` prefixed with `-MS`                                                                                                                      |
+| meta.profile\[0]                                 | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-MedicationStatement-1"`                                                                       |
+| identifier.system                                | `https:\\PSSAdaptor\{{practiceCode}}` where {{practiceCode}} is losing practice ODS Code.                                                                                  |
+| identifier.value                                 | `ehrSupplyAuthorise / id [@root]` prefixed with `-MS`                                                                                                                      |
```

Comment:
Suffixed.  I also think that markdown will want the [ to be escaped, but looks like it renders okay on GitHub, so not a biggie.

Reply:
Replaced Prefixed with Suffixed.  I think the [ doesn't need to be escaped as it's contain within a quotation block